### ISSUE TITLE
questions on config files and data augmentation

### ISSUE THREAD

**Issue by stephanecharette** [AlexeyAB/darknet#7114]
Alexey, I'd like to flesh out some of the entries on this page:  https://www.ccoderun.ca/darkmark/Configuration.html#ConfigNew

Specifically, some of the ones from the "new" table without any comment in the "notes" column.  Is there any text I can add or issue I should link to for these ones:

- cd53paspp-gamma.cfg
- csdarknet53-omega.cfg
- cspx-p7-mish-omega.cfg
- cspx-p7-mish.cfg
- cspx-p7-mish_hp.cfg
- efficientnet-lite3.cfg
- yolov4x-mish.cfg

In the same way, I would also like to update the entries on this page: 
 https://www.ccoderun.ca/darkmark/DataAugmentationMisc.html#Mosaic

- Is there some text I can add for mosaic, cutmix, or mixup?
- Are they all still supported?  I seem to often hear about mosaic, but not the other two.
- What can we add to that page to tell users when to use them, when not to use them, or what impact they have?
- Are the possible values just `0` and `1` for each option or does it get more complex?

Thanks!

**Comment by AlexeyAB** [comment]
- cd53paspp-gamma.cfg - detector, the same as yolov4.cfg, but with leaky instead of mish
- csdarknet53-omega.cfg - classifier, backbone for yolov4.cfg
- cspx-p7-mish-omega.cfg - classifier, backbone for cspx-p7-mish.cfg
- cspx-p7-mish.cfg - detector, yolov4-p7-large
- cspx-p7-mish_hp.cfg - detector, expiremental cfg-file
- efficientnet-lite3.cfg - classifier, EfficientNet-lite3
- yolov4x-mish.cfg - detector, something between `yolov4-csp` and `yolov4-p5`

----

- Is there some text I can add for mosaic, cutmix, or mixup? Only this: https://github.com/AlexeyAB/darknet/wiki/CFG-Parameters-in-the-%5Bnet%5D-section

- Are they all still supported?  I seem to often hear about mosaic, but not the other two. 
    * Mosaic usually is better than other two
    * The classifier supports all of them
    * Detector supports only mosaic

![image](https://user-images.githubusercontent.com/4096485/102013377-c834f280-3d60-11eb-93d6-681b1d0ad461.png)


- What can we add to that page to tell users when to use them, when not to use them, or what impact they have?

Use it always. You shouldn't use it only if your dataset is much larger and better than MS COCO.
https://arxiv.org/pdf/2004.10934.pdf

> Mosaic represents a new data augmentation method that
> mixes 4 training images. Thus 4 different contexts are
> mixed, while CutMix mixes only 2 input images. This allows detection of objects outside their normal context. In
> addition, batch normalization calculates activation statistics
> from 4 different images on each layer. This significantly
> reduces the need for a large mini-batch size.

- Are the possible values just `0` and `1` for each option or does it get more complex?

only 0 and 1
